00_organization/challenge_intro_slides.md

```diff
@@ -106,36 +119,45 @@ The challenge part:
 - [preCICE](https://precice.org/): Coupling library (Stuttgart)
 - [pyiron](https://pyiron.org/): Workflow manager in Python
 - [pyMOR](https://pymor.org/): MOR library in Python
+- [pyLife](https://pylife.readthedocs.io/en/stable/): fatigue of mechanical components in Python
 - [SU2](https://su2code.github.io/): CFD code in C++
 - [SUNDIALS](https://computing.llnl.gov/projects/sundials): Nonlinear solvers, ODEs
 - [TRILINOS](https://trilinos.github.io/): Collection of scientific software libraries, mainly solvers
 - [VisIt](https://visit-dav.github.io/visit-website/index.html): Scientific visualization software
-- more projects in [xSDK](https://xsdk.info/packages/) or [NumFOCUS](https://numfocus.org/sponsored-projects)
+- more projects in [xSDK](https://xsdk.info/packages/) or [NumFOCUS](https://numfocus.org/sponsored-projects) or [HiRSE Code Promotion](https://www.helmholtz-hirse.de/promo.html)
 - **Or your suggestion** (also agent-based or discrete event simulation software)
 
 ---
 
+## What Happens if Maintainers do not React in Time?
+
+- Give them time, contact early
+- Not your fault
+- Keep all communication public, then easy for us to review
+
+---
+
 ## How to Submit my Choice?
 
-- [https://gitlab-sim.informatik.uni-stuttgart.de/simulation-software-engineering-wite2425/challenge](https://gitlab-sim.informatik.uni-stuttgart.de/simulation-software-engineering-wite2425/challenge)
-- Comment in [issue #1](https://gitlab-sim.informatik.uni-stuttgart.de/simulation-software-engineering-wite2425/challenge/-/issues/1) till **23rd of October** (next Wednesday) evening (no FCFS)
+- [https://gitlab-sim.informatik.uni-stuttgart.de/simulation-software-engineering-wite2526/challenge](https://gitlab-sim.informatik.uni-stuttgart.de/simulation-software-engineering-wite2526/challenge)
+- Comment in [issue #1](https://gitlab-sim.informatik.uni-stuttgart.de/simulation-software-engineering-wite2526/challenge/-/issues/1) till **22rd of October** (next Wednesday) evening (no FCFS)
 - Priority list with at least three choices
 - If **not** on our suggestion list, write short paragraph what the software does and give links
 
 ---
 
 ## Role of Advisor
 
-- Benjamin, Frédéric, Gerasimos, or Ishaan
-- Use, for example, exercise blocks and time after lectures for discussions
+- Benjamin, Felix, Frédéric, Gerasimos, or Ishaan
+- Use, for example, exercise blocks and time before and after lectures for discussions
 - Share links etc. to issues and PRs (or tag us)
 
 ---
 
 ## Presentations
 
 - Length depends on number of students in course (maybe 5-10 mins)
-- Everybody has to present at least once
+- Everybody has to present at least once, better twice
```

Comment:
Any grade or other benefits / safeguards in presenting twice?

Reply:
Will change to "Everybody has to present once or twice (depending on number of students)